src/env/make_env.py

```diff
@@ -0,0 +1,102 @@
+import os
+from typing import Any, List, Optional
+
+import gym
+import numpy as np
+from pynktrombonegym.wrappers.action_by_acceleration import \
+    ActionByAcceleration
+from pynktrombonegym.wrappers.log1p_mel_spectrogram import Log1pMelSpectrogram
+
+from .array_action import ArrayAction
+from .array_voc_state import ArrayVocState
+from .normalize_action_range import NormalizeActionRange
+
+
+def make_env(dataset_dirs: List[Any], file_exts: Optional[List[str]] = None, 
+             action_scaler: Optional[float] = None, low: Optional[float] = None, high: Optional[float] = None, 
+             sample_rate: int = 44100, n_mels: int = 80, dtype: Any = np.float32) -> gym.Env:
+    """
+    Creates an environment instance from a list of audio dir paths.
+
+    Args:
+        dataset_dirs (List[Any]): A list of directory paths that contain audio files.
+        file_exts (Optional[List[str]]): A list of file extensions of audio files. Default is None.
+        action_scaler (Optional[float]): The scaling factor of action. Default is None.
+        low (Optional[float]): The lower limit of action range. Default is None.
+        high (Optional[float]): The upper limit of action range. Default is None.
+        sample_rate (int): The sample rate of audio files. Default is 44100.
+        n_mels (int): The number of mel bands to generate. Default is 80.
+        dtype (Any): The data type of the audio. Default is np.float32.
+        
+    Returns:
+        gym.Env: The created environment instance.
+    """
+    file_exts = file_exts or [".wav"]
+    files = create_file_list(dataset_dirs, file_exts)
+    
+    base_env_kwargs = {}
+    if n_mels is not None:
+        base_env_kwargs['n_mels'] = n_mels
+    if sample_rate is not None:
+        base_env_kwargs['sample_rate'] = sample_rate
+    if dtype is not None:
+        base_env_kwargs['dtype'] = dtype
+    base_env = Log1pMelSpectrogram(files, **base_env_kwargs)
+    
+    apply_wrappers_kwargs = {}
+    if action_scaler is not None:
+        apply_wrappers_kwargs['action_scaler'] = action_scaler
+    if low is not None:
+        apply_wrappers_kwargs['low'] = low
+    if high is not None:
+        apply_wrappers_kwargs['high'] = high
+    env = apply_wrappers(base_env, **apply_wrappers_kwargs)
```

Comment:
action_scalerは次のように指定してください。（加速度の積分計算に使用される値で、デフォルト値の設定は現実時間のスケールにするとうまくいくことがわかっています。)
```suggestion
    if action_scaler is None:
        action_scaler = base_env.generate_chunk / base_env.sample_rate
    env = apply_wrappers(base_env, action_scaler, low, high)
```